k8s/test/object_meta_matcher_test.go

```diff
@@ -0,0 +1,62 @@
+package k8s_test
+
+import (
+	"fmt"
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/gstruct"
+	"github.com/onsi/gomega/types"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ObjectMetaMatcher struct {
```

Comment:
Can we say `ObjectMetaDataMatcher` instead of `ObjectMetaMatcher` just to be a little more clear about what part of the generated YAML this is testing

Reply:
Turns out K8s calls it ObjectMeta so nevermind: https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#ObjectMeta